pubtools/_quay/signer_wrapper.py

```diff
@@ -176,11 +197,16 @@ def sign_containers(
         to_sign_entries = self._filter_to_sign(to_sign_entries)
         with redirect_stdout(io.StringIO()):
             run_in_parallel(
-                self.sign_container,
+                self.sign_container_chunk,
                 [
                     FData(args=x)
                     for x in zip(
-                        to_sign_entries,
+                        flatten(
+                            [
+                                list(grouper(group, chunk_size))
+                                for k, group in groupby(to_sign_entries, key=lambda x: x.repo)
```

Comment:
Why is it grouped by repo? Also, this code is almost incomprehensible. Can't it be written in a more understandable way?